pkg/cmd/cli/bug/bug.go

```diff
@@ -0,0 +1,144 @@
+/*
+Copyright 2018 the Heptio Ark contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package bug
+
+import (
+	"bytes"
+	"fmt"
+	"net/url"
+	"os/exec"
+	"runtime"
+	"text/template"
+
+	"github.com/heptio/ark/pkg/buildinfo"
+	"github.com/heptio/ark/pkg/cmd"
+	"github.com/spf13/cobra"
+)
+
+const (
+	linuxBrowserLaunchCmd = "xdg-open"
+	issueURL              = "https://github.com/heptio/ark/issues/new"
+	issueTemplate         = `
+**What steps did you take and what happened:**
+[A clear and concise description of what the bug is, and what commands you ran.)
+
+
+**What did you expect to happen:**
+
+
+**The output of the following commands will help us better understand what's going on**:
+(Pasting long output into a [GitHub gist](https://gist.github.com) or other pastebin is fine.)
+
+* ` + "`kubectl logs deployment/ark -n heptio-ark`" + `
+* ` + "`ark backup describe <backupname>` or `kubectl get backup/<backupname> -n heptio-ark -o yaml`" + `
+* ` + "`ark backup logs <backupname>`" + `
+* ` + "`ark restore describe <restorename>` or `kubectl get restore/<restorename> -n heptio-ark -o yaml`" + `
+* ` + "`ark restore logs <restorename>`" + `
+
+
+**Anything else you would like to add:**
+[Miscellaneous information that will assist in solving the issue.]
+
+
+**Environment:**
+
+- Ark version (use ` + "`ark version`" + `): {{.ArkVersion}}
+    - GitCommit: {{.GitCommit}}
+    - GitTreeState: {{.GitTreeState}}
+- Kubernetes version (use ` + "`kubectl version`" + `): 
```

Comment:
Any thoughts about running `kubectl version` and including it automatically here? I don't think it'd required, but I think it would make filing even easier.

Reply:
I think it makes sense to include, I wasn't sure if it was easier to use `kubectl` or the api.

Reply:
That's a good point. Ideally we'd have both the `kubectl` version and the server version, I think, so calling the executable makes sense to me. Open to other ideas here, though.

Reply:
`kubectl version` output added